android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -115,6 +124,21 @@ public Snapshot(View view) {
     }
     originXByParent = view.getLeft();
     originYByParent = view.getTop();
+    if (view.getBackground() != null) {
+      borderRadius = ((ReactViewBackgroundDrawable) view.getBackground()).getFullBorderRadius();
```

Comment:
1. Should we move this logic to `getViewBorderRadius` as well?
2. Does it make sense to check `instanceof ReactViewBackgroundDrawable` before casting?